documentation_builder/simulating.ipynb

```diff
@@ -758,13 +759,14 @@
       "PRODUCING REACTIONS -- Pyruvate\n",
       "-------------------------------\n",
       "  %             FLUX   RXN ID                        REACTION                       \n",
-      " 50.0%   6.13 ± 6.13      PYK                  adp_c + h_c + pep_c --> atp_c + pyr_c\n",
-      " 50.0%   9.76 ± 0.24   GLCpts                     glc__D_e + pep_c --> g6p_c + pyr_c\n",
+      " 85.0%   9.76 ± 0.24   GLCpts                     glc__D_e + pep_c --> g6p_c + pyr_c\n",
```

Comment:
True, I used half-spans +/- range/2 because its a familiar format. Probably better to make summary just have "MAX" and "MIN" columns when called with FVA.